tailwind.config.js

```diff
@@ -69,6 +69,23 @@ module.exports = {
           700: "#bf3989",
         },
       },
+      width: {
+        "150": "150px",
+        "200": "200px",
+        "250": "250px",
+        "300": "300px",
+        "450": "450px",
+        "500": "500px",
+        "600": "600px",
+        "750": "750px",
+        "860": "860px",
+        "12.5rem": "12.5rem",
```

Comment:
I think we should change this to something else? Possibly convert to px instead? I think 12.5 rem = 200px, not sure if this is alright

Reply:
Done :3